app/services/jwt-encryption.js

```diff
@@ -0,0 +1,12 @@
+const jwt = require('jwt-simple'),
+  config = require('../../config'),
+  { secret_key } = config.common.jwt;
+
+exports.generateToken = user => {
+  const tokenPayload = {
+    lastName: user.lastName,
+    firstName: user.firstName,
+    email: user.email
+  };
```

Comment:
Why are you saving first and last name? Also, take in consideration that this values will be deprecated